lib/app-express.js

```diff
@@ -149,27 +234,59 @@ module.exports = function (app) {
       }
 
       // check to see if org is a dev hub
-      const conn = new jsforce.Connection({
+      let connLocal = new jsforce.Connection({
```

Comment:
Should be `const`

Reply:
Also, why renamed?

Reply:
I stole conn and made it a global for the shared hub option.  connLocal is the original version (BYOhub)